var/spack/repos/builtin/packages/aml/package.py

```diff
@@ -23,3 +25,38 @@ class Aml(AutotoolsPackage):
     depends_on('autoconf', type='build')
     depends_on('automake', type='build')
     depends_on('libtool', type='build')
+
+    @run_after('install')
+    def cache_test_sources(self):
+        """Copy the example source files after the package is installed to an
+        install test subdirectory for use during `spack test run`."""
+        self.cache_extra_test_sources(['tests', join_path('include', 'config.h')])
+
+    def run_area_test(self):
+        """Run stand alone test: test_area"""
+
+        test_dir = join_path(self.install_test_root, 'tests', 'area')
```

Comment:
Please use the test stage copies for stand-one tests:

```suggestion
        test_dir = join_path(self.test_suite.current_test_cache_dir, 'tests', 'area')
```